beamer_control.ino

```diff
@@ -0,0 +1,104 @@
+#include <IRremote.h>
+
+int incomingByte = 0;
+
+IRsend irsend;
+
+void setup() {
+  Serial.begin(9600);
+}
+
+void loop() {
```

Comment:
Wouldn't it be more readable to have something like a hashmap here and just look up the codes from that?